src/App.svelte

```diff
@@ -63,27 +63,28 @@
 
 	async function addObscuroNetworkToMetaMask() {
 		console.log("Setting network to Obscuro")
-		const formattedChainId = "0x309"; // 777 in hex
-		try {
```

Comment:
I replaced the try/catch with handling the promise, since this approach only caught certain errors, and failed silently otherwise.